src/components/schedule-page/toolbar/toolbar.component.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from "react";
+import { ToolbarEditingComponent } from "./toolbar-editing.component";
+import { ToolbarViewingComponent } from "./toolbar-viewing.component";
+
+export function ToolbarComponent(): JSX.Element {
+  const [editMode, setEditMode] = useState<boolean>(false);
+
+  return (
+    <div className="toolbar-container">
+      {editMode ? (
+        <ToolbarEditingComponent editModeChange={setEditMode} />
+      ) : (
+        <ToolbarViewingComponent editModeChange={setEditMode} />
+      )}
+    </div>
+  );
+}
```

Comment:
Ja bym szczerze mówiąc nie implementowałbym tego w taki sposób ponieważ: 
1. My wprowadzamy komponenty który na razie nie ma żadnej logiki oprócz tego, że łączy dwa inne komponenty i przechowuje stan.
2. My przechowujemy stan na poziomie `toolbaru` chociaż od tego stanu zależy także to jak zachowuje się schedule, więc musi być conajmniej na poziomie `SchedulePage`
3. My nie możemy dodać teraz ładnych aanimacji, ponieważ teraz jest po prostu warunkowe renderowanie. 

Mi się wydaje, że o wiele sensowniejszem rozwiązaniem będzie rozdzielenie tego na dwa toolbary: `editing` i `viewing` i utworzenie dwóch osobnych podstron dla edycji i podglądu na których te toolbary będą się znajdować. ( nawigacja będzie robiona za pomocą `ReactRoute`)
To ma taką zaletę, że:
1. Pozbywamy się niepotrzebnego komponentu do lączenia `toolbarów`. 
2. Unikamy ifów podobnych do tego co jest tutaj ( bo zaszadniczo tryb edycji i tryb podglądu - to różne widoki. Wiem, że aktualnie są podobne, ale to nie zmienia tego faktu, że to może się zmienić i wtedy będziemy musieli dodawać te if'y o których pisałem wyżej. A to odrazu robi komponent nieczytelnym )
@VVlovsky ,  @pectom , co wy o tym sądzicie?

Reply:
Zgadzam się z @Qwebeck takie rozwiązanie wydaje się sensowniejsze. Fajnie gdybyś to wprowadził w tym PR.

Reply:
Dodana implementacja na `routing'u`